src/storage/src/hummock/store/state_store.rs

```diff
@@ -233,20 +233,36 @@ impl LocalStateStore for LocalHummockStorage {
 
     fn insert(&mut self, key: Bytes, new_val: Bytes, old_val: Option<Bytes>) -> StorageResult<()> {
         match old_val {
-            None => self.mem_table.insert(key, new_val)?,
-            Some(old_val) => self.mem_table.update(key, old_val, new_val)?,
+            None => {
+                self.mem_table.insert(key, new_val)?;
+            }
+            Some(old_val) => {
+                self.mem_table.update(key, old_val, new_val)?;
+            }
         };
+        self.stats
+            .mem_table_memory_size
+            .with_label_values(&[&self.table_id.to_string()])
+            .set(self.mem_table.mem_table_size as i64);
         Ok(())
     }
 
     fn delete(&mut self, key: Bytes, old_val: Bytes) -> StorageResult<()> {
+        self.stats
+            .mem_table_memory_size
+            .with_label_values(&[&self.table_id.to_string()])
+            .set(self.mem_table.mem_table_size as i64);
         Ok(self.mem_table.delete(key, old_val)?)
     }
 
     async fn flush(
         &mut self,
         delete_ranges: Vec<(Bound<Bytes>, Bound<Bytes>)>,
     ) -> StorageResult<usize> {
+        self.stats
+            .mem_table_memory_size
+            .with_label_values(&[&self.table_id.to_string()])
+            .set(0);
```

Comment:
We have multiple parallelism for a single table id. Therefore we shall do `inc` with the incremental value rather than `set`, which will overwrite the value written by other parallelism.

Reply:
Let‘s use `table_id` + `instance_id` as the metrics label.

Currently LocalHummock is not aware of `actor_id`, can we pass `actor_id` to it, and just use as the metrics label when creating, not store it?  This is because it is common to observe the size of `mem_table` when debugging. It would be useful to know the actor_id, while the `instacne_id` can only be used to distinguish the parallelism.